src/core/ready.js

```diff
@@ -11,7 +12,9 @@ var readyList = jQuery.Deferred();
 
 jQuery.fn.ready = function( fn ) {
 
-	readyList.then( fn );
+	readyList
+		.then( fn )
+		.catch( jQuery.readyException );
```

Comment:
I actually thought about that. 😄 I just thought the probability of someone registering a handler between running `$(fn)` and `fn` actually firing is small and current code is smaller. But OK, I can change that.